js/models/Order.js

```diff
@@ -0,0 +1,51 @@
+import BaseModel from './BaseModel';
+import _ from 'underscore';
+import app from '../app';
+
+export default class extends BaseModel {
+  defaults() {
+    return {
+      shipTo: '',
+      address: '',
+      city: '',
+      state: '',
+      postalCode: '',
+      countryCode: '',
+      addressNotes: '',
+      moderator: '',
+      items: [{
```

Comment:
Are you planning on keeping this flat, or are we going to use a nested collection?